mne/inverse_sparse/tests/test_mxne_inverse.py

```diff
@@ -41,7 +41,7 @@ def test_mxne_inverse():
     evoked.crop(tmin=-0.05, tmax=0.2)
 
     evoked_l21 = evoked.copy()
-    evoked_l21.crop(tmin=0.08, tmax=0.1)
+    evoked_l21.crop(tmin=0.081, tmax=0.1)
```

Comment:
The fragile nature of the `mxne` code came up again here. Including the one extra sample broke the test.